homeassistant/helpers/network.py

```diff
@@ -45,6 +45,18 @@ def host_ip() -> str | None:
             )
         )
 
+    def addon_url() -> str | None:
```

Comment:
Is this _just_ for URLs or anything routed via the supervisor ?

Reply:
I'm not aware of a use case for anything other then an addon using this helper but I suppose it could be for anything. I'm not sure we have a name for "generic supervised thing" though. `supervisor_component_url`? `docker_container_url`?